packages/mobile/src/transactions/transferFeedUtils.ts

```diff
@@ -74,8 +75,9 @@ function getRecipient(
       : recentTxRecipientsCache[phoneNumber]
 
     if (recipient) {
-      Object.assign(recipient, { address })
```

Comment:
Are we sure this isn't needed anymore?

Reply:
yes, because we only reach this part of the code if the recipient was retrieved from the `invitees` object, which doesn't keep track of addresses